corehq/apps/users/templates/users/web_users.html

```diff
@@ -17,14 +18,21 @@
       However, these users cannot submit forms from phones
       using the {{ cc_name }} Application.
     {% endblocktrans %}
-    {% if not request.is_view_only %}
+
       <div class="btn-toolbar">
```

Comment:
Should there be a check for `not request.is_view_only or can_access_all_locations` around the whole `.btn-toolbar`, or will it just be invisible if it's empty?

Reply:
Great q-- I just tested and the whole toolbar will be invisible if it's empty.